src/YMenuButton.cc

```diff
@@ -133,19 +134,85 @@ YMenuButton::findMenuItem( int wantedIndex, YItemConstIterator begin, YItemConst
     return 0;
 }
 
-
-void
-YMenuButton::resolveShortcutConflicts()
+static void resolveShortcutsConflictFlat(YItemIterator begin, YItemIterator end)
 {
-    // TO DO
-    // TO DO
-    // TO DO
+    bool used[ sizeof( char ) << 8 ];
+    for ( unsigned i=0; i < sizeof( char ) << 8; i++ )
```

Comment:
Now that you mention it, I have looked it up :) It is the other way around: char has at least 8 bits. The portable way is to `#include <limits.h>` and use `CHAR_BIT`
http://stackoverflow.com/questions/881894/is-char-guaranteed-to-be-exactly-8-bit-long